### ISSUE TITLE
Consider a way to mark an SVG as presentational

### ISSUE THREAD

**Issue by scottaohara** [w3c/svgwg#952]
As with HTML's img element, it seems to me there should be a way to mark an svg element as decorative without having to resort to using ARIA's aria-hidden=true or the role=none attribute on the element.

This is not to imply that those solutions don't work - and aren't known ways to achieve the effect of a decorative SVG.  But it just seems something that should be natively supported by SVG, rather than necessitating ARIA to achieve.

see some brief discussion on this when I had originally posted the thought for the ARIA wg to consider: https://github.com/w3c/aria/issues/2313
